### PR TITLE
Feature/editorial review for books

### PR DESCRIPTION
**Pull request by stuartjennings-up**
For this feature: https://trello.com/c/z12zIqWA/112-editorial-review-process-full-submission-full-implementation-required.

To be added to template Rua site for testing while I write some automated tests.

### CODE REVIEW

File: src/editor/urls.py [hunk 1]
```diff
@@ -24,13 +24,7 @@
                        url(r'^submission/(?P<submission_id>\d+)/review/round/(?P<round_number>\d+)/withdraw/(?P<review_id>\d+)/$', 'editor.views.editor_review_round_withdraw', name='editor_review_round_withdraw'),
 
                        # Editorial
-                       url(r'^review/submission/(?P<submission_id>\d+)/editorial/(?P<review_id>[-\w./]+)/delete/$', 'editor.views.editorial_review_delete', name='editorial_review_delete'),
```
Comment (stuartjennings-up) [inline src/editor/urls.py:27]: Removing urls for old editorial review system

File: src/editor/views.py [hunk 2]
```diff
@@ -523,71 +555,6 @@ def editor_change_revision_due_date(request, submission_id, revision_id):
 
 
 @is_book_editor
-def editorial_review_view(request, submission_id, review_id):
```
Comment (stuartjennings-up) [inline src/editor/views.py:526]: Removing views for old editorial review system

File: src/editor/logic.py [hunk 3]
```diff
@@ -282,7 +282,7 @@ def send_editorial_review_update(book, review_assignment, email_text, sender, at
         'sender': sender,
```
Comment (stuartjennings-up) [inline src/editor/logic.py:282]: These long lines upset me too but changing these would mean changing everything in that document which would also upset me.

**Review by fradeve — APPROVED** [review]
Half of this is templates, and we need some serious effort on PEP8 (probably for later), but LGTM :+1:

**Pull request merged**